src/api/Cargo.toml

```diff
@@ -7,10 +7,11 @@ documentation.workspace = true
 edition.workspace = true
 
 [features]
-default = ["native-as", "rustls"]
-native-as = ["attestation-service/default"]
-native-as-no-verifier = ["attestation-service/rvps-native"]
-grpc-as = ["tonic", "tonic-build", "prost"]
+default = ["coco-as-builtin", "rustls"]
+coco-as = []
+coco-as-builtin = ["coco-as", "attestation-service/default"]
+coco-as-builtin-no-verifier = ["coco-as", "attestation-service/rvps-native"]
```

Comment:
A question not related to this PR. Why do we need a "no-verifier" feature? If it is for test, we could implement a mock-ed object of trait `Attest`

cc @jialez0

Reply:
The `no-verifier` is really for testing purposes, so that one can build the project without pulling all the tdx/sgx dependencies.
I agree a mocked implementation of the Attest trait would make more sense, I like that.

Reply:
Great. I'll work on this then